### ISSUE TITLE
Widescreen token won't work

### ISSUE THREAD

**Issue by martymcflysenior** [uswds/uswds#3145]
## Description
Using the _uswds-theme-components.scss, I'm trying to set my site to widescreen without success. 
Updated $theme-header-max-width and $theme-banner-max-width to 'widescreen', but I'm still seeing  max-width: 64rem in .usa-banner__inner, region region-header usa-navbar and .usa-nav__inner. 

## Steps to reproduce the issue

1.  comment out @import 'uswds'; in style.scss
2.  import all packages listed starting on line 25-52
3.  import utilities on line 56
4.  import variables @import "_variables.scss";
5. Updated $theme-header-max-width and $theme-banner-max-width to 'widescreen', but I'm still seeing  max-width: 64rem in .usa-banner__inner, region region-header usa-navbar and .usa-nav__inner.

**Comment by thisisdano** [comment]
This should be fixed in USWDS `2.3.0`: https://github.com/uswds/uswds/releases/tag/v2.3.0
Let me know if it's fixed for you. Thanks for your patience.

**Comment by mejiaj** [comment]
Fix in `2.3.0` is confirmed.

Tested using:
- [x]  `uswds-sandbox`

@thisisdano, I tried the same approach in uswds locally and it wasn't updating for me. 

<details><summary>styles.scss</summary>

```css
/*
* * * * * ==============================
* * * * * ==============================
* * * * * ==============================
* * * * * ==============================
========================================
========================================
========================================
*/

// -------------------------------------
// Import individual theme settings

@import "uswds-theme-general";
@import "uswds-theme-typography";
@import "uswds-theme-spacing";
@import "uswds-theme-color";
@import "uswds-theme-utilities";

// components import needs to be last
@import "uswds-theme-components";

// -------------------------------------
// Import individual USWDS packages...
// See https://designsystem.digital.gov/components/

// First import required and global packages...
@import 'packages/required';
@import 'packages/global';

// Then import individual component packages...
// @import 'packages/form-controls';
// @import 'packages/form-templates';
// @import 'packages/layout-grid';
// @import 'packages/layout-grid';
// @import 'packages/typography';
// @import 'packages/validation';
// @import 'packages/usa-accordion';
// @import 'packages/usa-alert';
@import 'packages/usa-banner';
// @import 'packages/usa-button';
// @import 'packages/usa-checklist';
// @import 'packages/usa-footer';
@import 'packages/usa-header';
// @import 'packages/usa-hero';
// @import 'packages/usa-media-block';
// @import 'packages/usa-megamenu';
@import 'packages/usa-nav-container';
// @import 'packages/usa-nav';
// @import 'packages/usa-navbar';
// @import 'packages/usa-search';
// @import 'packages/usa-sidenav';
// @import 'packages/usa-skipnav';
// @import 'packages/usa-table';
// @import 'packages/usa-tag';

// or package sets...
// @import 'packages/uswds-components';
// @import 'packages/uswds-utilities';

// -------------------------------------
// ...or import the complete USWDS kit

@import "uswds";

// -------------------------------------
// Import theme custom styles

@import "uswds-theme-custom-styles";
```
</details>

<details><summary>_uswds-theme-components.scss (only relevant lines)</summary>

```css
// Banner
$theme-banner-max-width: "widescreen";

// Header
$theme-header-max-width: "widescreen";
```
</details>
